src/background.ts

```diff
@@ -59,16 +59,16 @@ chrome.runtime.onMessage.addListener(function (request, sender, callback) {
     switch(request.message) {
         case "openConfig":
             chrome.tabs.create({url: chrome.runtime.getURL('options/options.html' + (request.hash ? '#' + request.hash : ''))});
-            return;
+            return true;
         case "openHelp":
             chrome.tabs.create({url: chrome.runtime.getURL('help/index.html')});
-            return;
+            return true;
         case "openUpsell":
             chrome.tabs.create({url: chrome.runtime.getURL('upsell/index.html')});
-            return;
+            return true;
         case "openPage":
             chrome.tabs.create({url: chrome.runtime.getURL(request.url)});
-            return;
+            return true;
```

Comment:
I think these should all be false, otherwise the port stays open

Reply:
true/false is for async or sync

Reply:
True = async